.editorconfig

```diff
@@ -10,9 +10,14 @@ insert_final_newline = true
 indent_style = tab
 indent_size = 4
 
-[{*.json,*kml,*.md}]
+[{.*,*.json,*.md}]
 indent_style = space
 indent_size = 2
 
-[*.csv]
+[site/includes/sitemenureplace.hbs]
```

Comment:
@nschonni Would it be a better practice to only specify the file's name (instead of its full path) in this kind of situation?

The repo doesn't contain any other files with the same names as the ones I'm targeting here and below.

Reply:
I think the indentation should be fixed in the file rather than an exception. The file https://github.com/wet-boew/wet-boew/blob/f5f33e0240649fe22fd12f28978113841762bffa/site/includes/sitemenu.hbs#L1 just needs to have that leading space instead

Reply:
Ok. I'll remove the indent rules from that part (although the newline override will remain).

As far as modifying sitemenu.hbs goes... I'll do it in an incoming PR (whose goal is to bring the entire repo in line with this PR's .editorconfig rules). Should have it out later today or tomorrow.